src/components/Input.js

```diff
@@ -0,0 +1,32 @@
+import styled from 'styled-components/macro'
+import PropTypes from 'prop-types'
+
+function Input({value, onChange, placeholderText}) {
```

Comment:
What is the idea behind making the input field a separate component? Maybe (/Probably :) I don't see the full potential but it's basically the same amount of code / work to use and integrate this component into another one (in this case in the Form-component) as just build a regular form with an input field. What am I missing?